.github/workflows/python_uniffi_ci_cd.yml

```diff
@@ -210,7 +210,7 @@ jobs:
         env:
           RUN: |
             cd /workspace/packages/python/${{ inputs.crate }}
-            python${{ env.PYTHON_VERSION }} -m maturin build --release --target ${{ env.TARGET }} --compatibility ${{ env.MANYLINUX }} --features ffi_uniffi
+            python${{ env.PYTHON_VERSION }} -m maturin build --release --target ${{ env.TARGET }} --compatibility ${{ env.MANYLINUX }} --no-default-features --features ffi_uniffi
```

Comment:
Just to make sure I understand, the `--no-default-features` flag is required here and many other places because the testing feature? Can we simply not make the test-related features default?

Reply:
Good question. Yeah specifically the test data generation capability, which leverages `algokit_transact/test_utils`.
In answering this I realised I made the assumption that the test_utils features would be statically linked into the built transact ffi package binaries, however upon testing it doesn't appear to be true 😄

Enabling the feature by default actually reduces the wasm binary output size, which is interesting.

I'll update based on your suggestion.